### ISSUE TITLE
GameGuru MAX - Request - Load Screen Hints

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#5052]
Many games now, knowing that it takes good minute to load a level, display hints about how to play and do things in the game on the level load screen. These hints cycle - so they stay up long enough to be read, then cycle to the next one, then the next ... as long as the level is loading. They are random, too. It's a good way to make the level load time seem shorter than it is by giving the player something to read that actually helps them play the game.

Therefore, I'd like to request that MAX have the ability to display a screen while loading a level (a unique screen, depending on where you are in the game) that can draw from a list of "hints" that can be displayed wherever the developer wants them (normally toward the bottom-center of the screen).

**Comment by mav3r1ck1981** [comment]
yes please!!! wouldnt mind some form of picture that could be displayed from a "pool" in the same manner as well

**Comment by MonkeyFrogStudio** [comment]
@mav3r1ck1981 Absolutely - a txt document with all the blurbs/hints and a folder that could be pointed to with images to draw from ... and some convention that ties a blurb/hint to an image.

**Comment by MonkeyFrogStudio** [comment]
Some random examples pulled from the net:

![2c071e_7392244](https://github.com/TheGameCreators/GameGuruRepo/assets/36735507/6fd8510d-5047-4618-9c3a-11d25f949e48)

![b44ce1f5ce4985fd4376bfb1f658ad9039582136](https://github.com/TheGameCreators/GameGuruRepo/assets/36735507/dbee1bfb-bf14-4fc8-8283-d8dea846b476)

![clash-of-clans-loading-screen-hints](https://github.com/TheGameCreators/GameGuruRepo/assets/36735507/213d6464-3424-44d6-8ffb-2e7f9d232ed8)

![e131a04b06f6445f80d871e5d3a860d7a8eed396](https://github.com/TheGameCreators/GameGuruRepo/assets/36735507/894f8513-22cb-47e5-9e53-9dd821eb78a1)

![F4OdL](https://github.com/TheGameCreators/GameGuruRepo/assets/36735507/42f01b20-1059-4679-83bb-75097b233a87)

![h21QC](https://github.com/TheGameCreators/GameGuruRepo/assets/36735507/e28fdae4-4fee-48ef-be72-f1dfb797116d)

![kVg3DnfseBDqvJedZgsZF7](https://github.com/TheGameCreators/GameGuruRepo/assets/36735507/c396a880-226b-4452-92aa-b28a9f09c295)

**Comment by extremestrategy** [comment]
I like this idea.
